docs/src/pages/components/date-field.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: DateField
+group: Forms
+edit: https://github.com/marigold-ui/marigold/blob/main/docs/src/pages/components/container.mdx
```

Comment:
the link is right now set to container.mdx ist should be to this file: .....date-field.mdx

Reply:
also github link is missing (github link is where the component lies. edit link is where the mdx file lie.